src/proto/tbots_software_msgs.proto

```diff
@@ -16,5 +17,7 @@ message PrimitiveSet
     // Robot ID to RadioPrimitive map
     map<uint32, Primitive> robot_primitives = 3;
 
+    map<uint32, Angle> robot_orientations = 5;
```

Comment:
Im having a hard time understanding why this field is needed when `orientation` also exists within the `Primitive` msg.

Reply:
@mkhlb this is a good question